config.go

```diff
@@ -44,6 +44,12 @@ const (
 
 	SettingDevicemonitorAddr        = "devicemonitor_addr"
 	SettingDevicemonitorAddrDefault = "http://mender-devicemonitor:8080"
+
+	SettingEnableReporting        = "enable_reporting"
+	SettingEnableReportingDefault = false
+
+	SettingOrchestratorAddr        = "orchestrator_addr"
+	SettingOrchestratorAddrDefault = "http://mender-workflows-server:8080"
```

Comment:
enabling reporting support without workflows URL is a config error, could we check this on config load in the validation phase?

Reply:
We don't do this kind of validation in the other microservices. There was an error, though, and the default was not correctly set. I updated it.